apps/website/netlify/functions/strangler.ts

```diff
@@ -0,0 +1,18 @@
+import { createStrangler } from '@amazeelabs/strangler-netlify';
+import fs from 'fs';
+
+export const handler = createStrangler(
+  [
+    {
+      // For Standard drupal redirects, we are interested in any
+      // url (therefore no urlFilter) and only in redirects (301, 302).
+      url:
+        process.env.DRUPAL_INTERNAL_URL ||
```

Comment:
Where are the serverless functions executed? On Netlify? If yes, then `DRUPAL_INTERNAL_URL` should be unreachable... Or do I miss something?